test/Campaign.test.js

```diff
@@ -0,0 +1,85 @@
+const assert = require("assert");
+const CampaignFactory = artifacts.require("CampaignFactory");
+const Campaign = artifacts.require("Campaign");
+
```

Comment:
This is the updated test file and should give an idea of how the interactions with contracts will look.

Reply:
It may be worth going over this file as a whole team to understand how interactions with the blockchain are currently working.